regolith/helpers/a_proposalhelper.py

```diff
@@ -28,70 +23,57 @@ def subparser(subpi):
     subpi.add_argument("name", help="A short but unique name for the proposal",
                        default=None
                        )
-    subpi.add_argument("-amt", "--amount", help="value of award",
+    subpi.add_argument("amount", help="value of award",
                        default=None
                        )
-    subpi.add_argument("-a", "--authors",
-                       help="Other investigator names"
+    subpi.add_argument("authors", help="Other investigator names"
                        )
-    subpi.add_argument("--begin_day", nargs="+",
-                       help="The start day for the proposed grant in format YYYY-MM-DD."
+    subpi.add_argument("currency", help="Currency in which amount is specified."
+                                        " Typically $ or USD"
                        )
-    subpi.add_argument("--begin_month", nargs="+",
-                       help="The start month for the proposed grant in format YYYY-MM-DD."
+    subpi.add_argument("due_date", help="The due date for the proposal in format YYYY-MM-DD."
                        )
-    subpi.add_argument("--begin_year", nargs="+",
-                       help="The start year for the proposed grant in format YYYY-MM-DD."
+    subpi.add_argument("duration", help="Duration of proposal in years"
                        )
-    subpi.add_argument("--end_date", nargs="+",
-                       help="The end date for the proposed grant in format YYYY-MM-DD."
+    subpi.add_argument("submitted_date", help="Day on which the proposal is submitted "
+                                              "in format YYYY-MM-DD"
                        )
-    subpi.add_argument("--call_for_proposals", nargs="+",
-                       help="Url for call for proposals"
+    subpi.add_argument("pi", help="Name of principal investigator"
```

Comment:
please make optional.  by default have it find the group pi name from the rc.  Please see how this is done in the projectum adder

Reply:
Will do! Just to clarify, the projectum adder displays the `pi_id` whereas here I am trying to display the `pi` name. Should I change my adder to display `pi_id`? 
Alternately, I have looked the `get_pi_id(rc)` function in `regolith/tools.py` and I can write another function `get_pi_name(rc)` that returns the `pi` name, if that's a better option. Unless, do we have a function in regolith that returns the name of a group member given their id?